test/extended/router/scoped.go

```diff
@@ -335,3 +337,126 @@ func ingressForName(r *routev1.Route, name string) *routev1.RouteIngress {
 	}
 	return nil
 }
+
+const overridePemData = `
```

Comment:
I checked, and `defaultPemData == overridePemData`, so I wouldn't object to deleting `overridePemData` and using `defaultPemData` instead.

Reply:
Done